notifier/import_map.json

```diff
@@ -0,0 +1,11 @@
+{
+  "imports": {
+    "envalid": "https://deno.land/x/envalid@0.1.2/mod.ts",
+    "grammy": "https://deno.land/x/grammy@v1.11.2/mod.ts",
+    "oak": "https://deno.land/x/oak@v11.1.0/mod.ts",
+    "oak/": "https://deno.land/x/oak@v11.1.0/",
+    "octokit/": "https://raw.githubusercontent.com/octokit/webhooks-methods.js/v2.0.0/src/",
+    "postgres": "https://deno.land/x/postgres@v0.17.0/mod.ts",
+    "std/": "https://deno.land/std@0.164.0/"
+  }
+}
```

Comment:
Consider being consistent here by never including the file path to entrypoints in the import map.

Reply:
Do you by entrypoint mean "/"?

Reply:
If I don't do "std/", neither "testing/", I will waste minutes in thinking of a good name for "testing/asserts.ts".

Reply:
I mean that `/mod.ts` could be moved to the import statements, rather than being hard-coded here.